src/middleware/routers/beneficiaries.js

```diff
@@ -17,17 +17,25 @@ const passport = require('passport');
 const {
   router: { beneficiaries },
 } = require('../../controllers');
+const { Forbidden } = require('../../utils/errors');
 
 /**
  * Routing for beneficiaries endpoints (/v1/beneficiaries)
  */
-// GET /v1/beneficiaries
 
-router.get(
-  '/',
-  passport.authenticate('jwt', { session: false }),
-  beneficiaries.getAll
-);
+// GET /v1/beneficiaries
+function isLoggedIn(req, res, next) {
+  passport.authenticate('jwt', { session: false }, (err, user) => {
+    // err === 'user not found' || user === false || user === undefined
+    if (err || !user) {
+      return next(new Forbidden(' You are forbidden to enter.'));
+    }
+
+    return next();
+  })(req, res, next);
+}
```

Comment:
Let's move this function to its own file under middleware, because it'll need to be reused by the other routes